src/adapter.ts

```diff
@@ -54,11 +59,11 @@ export class SequelizeAdapter implements Adapter {
 
   private beforeAddModel(): void {
     const options = getOptions(CasbinRule.prototype);
-    if (options != undefined){
-      if (this.hookModelOption != undefined){
-        this.hookModelOption(options)
+    if (options != undefined) {
+      if (this.hookModelOption != undefined) {
+        this.hookModelOption = options;
```

Comment:
It does not use the `this.hookModelOption` which is user passed.
You should change `options` through `this.hookModelOption` and then set the options.

Reply:
It looks better to merge objects, so like:
```
const options = {...getOptions(CasbinRule.prototype), ...this.hookModelOption}
```